README.md

```diff
@@ -1,3 +1,22 @@
+# Project Status
+Releases are currently only created as they are needed. We are actively seeking contributors to take ownership of creating new releases.
+
+An alternative approach to the Indy SDK is being developed which divides LibIndy into multiple shared libraries which will be part of the Indy and Aries projects:
+
+* a verifiable data registry for communication with Indy ledgers,
+* an encrypted storage service for Hyperledger Aries agents,
+* a credential exchange library,
+* and a library of shared Rust functions.
+
+Progress is recorded in the notes for the [Indy Contributors Meeting](https://wiki.hyperledger.org/display/indy/Indy+Contributors+Meeting).
+
+[Indy Node](https://github.com/hyperledger/indy-node) will continue to be maintained and improved.
```

Comment:
Please reword to:

'Hyperledger Indy's ledger implementation, [Indy Node](https://github.com/hyperledger/indy-node) and related codebases, continue to be maintained and improved.'

I'd suggest moving this up to right below the line about "New Indy-SDK releases..."